### PR TITLE
Index js refactor

### PR DESCRIPTION
**Pull request by karinamzalez**
- refactor collision out of index.js
- ALL 60 TESTS PASSING 
- note to add junkFood can take attributes upon instantiation tests

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -45,6 +48,7 @@ startButton.addEventListener('click', function(){
   window.setInterval(createHealthFood, 1500);
 });
 
+
```
Comment (deborahleehamel) [inline lib/index.js:51]: extra spaces we can delete

File: lib/tray-collision.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+function recordScore(foodsArray, food, instance){
+  var newScore = +food.score;
+  foodsArray.shift(food);
+  instance.total += newScore;
+  document.getElementById('score-game').innerHTML = "Score: " + instance.total;
+}
+
+TrayCollision.prototype.gameOver = function(instance, reqAnimID){
```
Comment (deborahleehamel) [inline lib/tray-collision.js:31]: Looks like we still need to figure out clearing the game after game over.

File: test/tray-collision-test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const chai = require('chai');
+const assert = chai.assert;
+const TrayCollision = require('../lib/tray-collision');
+
+describe('TrayCollision', function(){
```
Comment (deborahleehamel) [inline test/tray-collision-test.js:5]: Nice! Adding tests as you go.

File: lib/tray-collision.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    var canvas = document.getElementById('game');
+    canvas.style.display = "none";
+    var startButton = document.getElementById('start');
+    startButton.style.display = "block";
+    document.getElementById('score-game').innerHTML = " ";
+    instance.total = 0;
+    return window.cancelAnimationFrame(reqAnimID);
+  }
+};
+
```
Comment (chompasina) [inline lib/tray-collision.js:43]: Great refactor pulling out dependencies to separate modules!

**Comment by deborahleehamel** [comment]
I will add the junkFood test follow up note to our waffle.

**Comment by chompasina** [comment]
Looks good to me.

**Pull request merged**